lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -317,11 +317,11 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 12, 7),
-        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 12, 7),
+        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 13, 8),
+        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 13, 8),
         # split mongo: 3 queries, regardless of thread response size.
-        (ModuleStoreEnum.Type.split, 1, 3, 3, 12, 7),
-        (ModuleStoreEnum.Type.split, 50, 3, 3, 12, 7),
+        (ModuleStoreEnum.Type.split, 1, 3, 3, 13, 8),
+        (ModuleStoreEnum.Type.split, 50, 3, 3, 13, 8),
```

Comment:
I realize that in the PR we won't see the counts going down as there is no 'before' result, but it would be good to run the tests against master just so we know what the improvement looks like.